### ISSUE TITLE
Add GNU/kFreeBSD support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#38]
```
Hi!

The attached patch will allow leveldb to compile on kFreeBSD platforms.

Regards,
```

---

Original issue reported on code.google.com by `quadris...@gmail.com` on 5 Sep 2011 at 7:56

Attachments:
- [1002-kfreebsd.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-38/comment-0/1002-kfreebsd.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks! Do you know if the GNU/kFreeBSD case could somehow be merged with the 
FreeBSD case of the switch statement?
```

---

Original comment by `ga...@google.com` on 23 Sep 2011 at 10:31

**Comment by GoogleCodeExporter** [comment]
```
This is now fixed in revision 26db4d971a15
```

---

Original comment by `ga...@google.com` on 28 Sep 2011 at 10:08
- Changed state: **Fixed**
